BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -396,6 +396,25 @@ private IEnumerator PilotLogic()
                     }
                 }
             }
+            else if (hasPropulsion && (currentCommand == PilotCommands.FlyTo || currentCommand == PilotCommands.Follow))
+            {
+                // We have been given a command from the WingCommander to fly/follow in a general direction
+                // Burn for commandedBurn length, coast for 2x commandedBurn length
+                SetStatus("Maneuvering " + (currentCommand == PilotCommands.FlyTo ? "(Commanded Position)" : "(Following)"));
+                float timer = 0;
+
+                while (UnderTimeLimit(commandedBurn * 3f) && (currentCommand == PilotCommands.FlyTo || currentCommand == PilotCommands.Follow))
+                {
+                    if (currentCommand == PilotCommands.FlyTo)
+                        fc.attitude = (assignedPositionGeo - vessel.transform.position).normalized;
```

Comment:
Shouldn't this be `assignedPositionWorld` instead of `assignedPositionGeo`?

Reply:
Um, no, `assignedPositionGeo` is using `WorldPositionToGeoCoords` which gives latitude, longitude and altitude coordinates. `assignedPositionWorld` is using `GetWorldSurfacePostion` which gives the 3D space position for those coordinates relative to the current floating origin.
E.g., see lines 128—130 of VesselSpawnerBase.cs.